NOTICE.txt

```diff
@@ -1,3 +1,3 @@
 Carbon
 
-Copyright 2012 - 2015 Aaron Jensen
\ No newline at end of file
+Aaron Jensen
```

Comment:
Should this be `Copyright Aaron Jensen and WebMD Health Services` ?

Reply:
Yup. Fixed.